tests/cloud_test_logic/cloud_test_index.py

```diff
@@ -110,5 +114,51 @@ class CloudTestIndex(str, Enum):
         "imagePreprocessing": {
             "patchMethod": "simple",
         }
-    }
+    },
+    CloudTestIndex.structured_text: {
```

Comment:
There's a duplicate of structured text and unstructured no model, probably because I rearranged the order in my PR.

Reply:
removed duplicate